quantinuum-hugr/src/types/type_param.rs

```diff
@@ -246,6 +251,12 @@ impl TypeArg {
     }
 }
 
+impl TypeArgVariable {
```

Comment:
There's no docstring here, why isn't CI catching this?

Reply:
Uh, seems I missed setting `lints.workspace = true` in the package's Cargo.toml when I created the workspace...

I can fix it in a separate PR, or you can add
```toml
[lints]
workspace = true
```
in `quantinuum-hugr/Cargo.toml`.

Reply:
See #876